src/components/Radio/Radio.stories.js

```diff
@@ -136,6 +146,69 @@ class FlatRadioGroup extends React.Component {
   }
 }
 
+class MenuRadioGroup extends React.Component {
```

Comment:
This could be a functional component using hooks! _(But if you don't wanna that's also fine, class components still work just fine haha)_

Reply:
Yeah I just copied what I already. Let's leave it like this since it doesn't break anything

Reply:
cool!